core/src/main/scala/org/apache/spark/deploy/PythonRunner.scala

```diff
@@ -92,15 +92,16 @@ object PythonRunner {
       // see https://github.com/numpy/numpy/issues/10455
       sparkConf.getOption("spark.driver.cores").foreach(env.put("OMP_NUM_THREADS", _))
     }
-    builder.redirectErrorStream(true) // Ugly but needed for stdout and stderr to synchronize
```

Comment:
Why is this not needed anymore?

Reply:
since we do this in `ProcessRedirectThread`